pkg/security/secl/model/model_unix.go

```diff
@@ -61,7 +61,8 @@ type Event struct {
 	LoginUIDWrite LoginUIDWriteEvent `field:"-"`
 
 	// network syscalls
-	Bind BindEvent `field:"bind" event:"bind"` // [7.37] [Network] A bind was executed
+	Bind    BindEvent    `field:"bind" event:"bind"`                      // [7.37] [Network] A bind was executed
+	Connect ConnectEvent `field:"connect;connect.server" event:"connect"` // [7.60] [Network] A connect was executed
```

Comment:
I think the `server` alias should be put within the `ConnectEvent` struct to avoid having a `connect.server.retval` field which doesn't make sense.
Something like:
```
 type SockAddr struct {
    IPPortContext // embed
    Family `field:"family"`
}

type ConnectEvent struct {
   SyscallEvent
   SockAddr `field:"addr;server.addr"`
   Protocol
}
```

Reply:
But this can probably be addressed in a followup PR